### ISSUE TITLE
Input and output buses

### ISSUE THREAD

**Issue by Bobingstern** [UnsignedArduino/LogicalSimulator#265]
Basically, there should be a type of input called a "sized" input (name work in progress) that is basically just a bunch of inputs stacked into one that you can change in a bubble ui or something. For example, if you have an IC for 2 8-bit numbers and 1 8-bit output, you should just have 2 "sized" inputs and 1 "sized" output (they are just a bunch of i/o squashed into 1)  to save space. The switches should also have a "sized".
Basically, input and output buses. Would be very useful for CPU designing

**Comment by Bobingstern** [comment]
Yes this is exactly what I had in mind
